cypress/e2e/e2e.cy.js

```diff
@@ -0,0 +1,105 @@
+describe('Teste E2E EngageSphere', () => {
+    beforeEach(() => {
+        cy.setCookie('cookieConsent', 'accepted')
+        cy.visit('/')
+    })
+
+    it('Retorna à lista de clientes ao clicar no botão "Voltar"', () => {
+        cy.contains('button', 'View').should('be.visible').click()
```

Comment:
Para simplificar, você pode remover a verificação intermediária.

```suggestion
        cy.contains('button', 'View').click()
```

O Cypress já espera por elementos estarem visíveis antes de clicar neles, portanto, use tal abordagem somente quando necessário.

O mesmo vale para a linha abaixo.